dev/com.ibm.ws.grpc.client/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -0,0 +1,49 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# -------------------------------------------------------------------------------------------------
+#CMVCPATHNAME com.ibm.ws.grpc.client/resources/OSGI-INF/l10n/metatype.properties
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+
+
+clientConfig=GRPC Client Properties
+clientConfig.desc=Configuration properties to be applied to gRPC targets that match the specified URI when they are constructed
+
+target=Remote target
+target.desc=The remote gRPC target URI, with wildcard support.
+
+headersToPropagate=HTTP header names to propagate
+headersToPropagate.desc=HTTP header names to propagate from an inbound request to an outbound call.
+
+clientInterceptors=GRPC client Interceptors
+clientInterceptors.desc=A list of fully qualified class names for gRPC client interceptor classes.
+
+authnToken=Authorization token
+authnToken.desc=The type of authorization token to propagate.  Currently this must be "jwt" or "oauth".
+
+enableKeepAlive=Keep alive
```

Comment:
this is news to me but IBM terminology specifies keepalive, one word. Here's the guidance-

keepalive
adjective
Pertaining to the amount of time that elapses before a message is sent to verify a connection.

noun
A TCP/IP mechanism where a small packet is sent across the network at predefined intervals to determine whether the socket is still working correctly.

If this description applies to the function, update all instances to a single-word term

Reply:
I agree, this should be one word.  I'll update.